action.yml

```diff
@@ -1,6 +1,6 @@
-name: 'ROS 2 CI'
+name: 'mjlee111 ROS 2 CI'
 description: 'Continuous integration for ROS 2 project'
-author: "ICHIRO ITS"
+author: "mjlee111"
```

Comment:
Can you keep the original name and author?

Reply:
Sorry for the inconvenience. I will PR again after reverting those changes.